lauth/spec/requests/delegated_spec.rb

```diff
@@ -0,0 +1,69 @@
+RSpec.describe "/authorized delegation", type: [:request, :database] do
+  let!(:user) { Factory[:user, userid: "lauth-allowed"] }
+  before(:each) do
+    setup_coll("secret", has_grant: false, pub: false)
+    setup_coll("public", has_grant: false, pub: true)
+    setup_coll("extra",  has_grant: true,  pub: false)
+    setup_coll("both",   has_grant: true,  pub: true)
+    target = setup_coll("target", has_grant: true,  pub: false)
+    Factory[:location, dlpsPath: "/delegated", collection: target]
+  end
+
+  context "when authorized" do
```

Comment:
what is this actual scenario? maybe "with an authorized, delegated collection" or so?